python-shell/src/gafferpy_examples/__init__.py

```diff
@@ -0,0 +1,11 @@
+__version__ = "2.0.0-alpha-0.3"
+
+__title__ = "gafferpy"
+__description__ = "Gaffer Python Shell"
+__uri__ = "https://github.com/gchq/gafferpy-tools"
```

Comment:
Does this actually go anywhere?

Reply:
It is used in the gaffer module, but here it is not. I do agree it's a bit weird, but @GCHQDeveloper314 is adding a copyright checker that requires non-empty files so we should figure out something useful to put in these, even if it's a copyright header and some explanation of what the module is

Reply:
In this instance I mean the URI seems to lead to a dead end. I'm guessing this was merged into gaffer-tools a while ago and this wasn't updated.

Reply:
Another good point... we should change that yes, will fix in this PR